src/main/java/com/couchbase/lite/store/SQLiteStore.java

```diff
@@ -289,6 +289,9 @@ else if (e.getCode() == SQLException.SQLITE_ENCRYPTION_NOTAVAILABLE)
             }
 
             if (isNew)
+                setInfo("pruned", "true"); // See -compact: for explanation
+
+            if (isNew)
```

Comment:
https://github.com/couchbase/couchbase-lite-ios/blob/c32147bd6e83c4c4b24b9159ff1d52ca9d0cbaf0/Source/CBL_SQLiteStorage.m#L464-L465
```objective-c
    if (isNew)
        [self setInfo: @"true" forKey: @"pruned"];  // See -compact: for explanation
```
isNew is fine.